pkg/manager/manager.go

```diff
@@ -15,9 +17,24 @@ import (
 )
 
 const (
-	vlanUplinkLockFile = "/tmp/accelerated-bridge-cni.lock"
+	vlanUplinkLockFile = "/var/lib/cni/accelerated-bridge/vlan-uplink.lock"
 )
 
+type VlanLock interface {
```

Comment:
i dont think we care if the lock is used for vlan here.

maybe rename to just `Lock` ? or IPCLock ?
also please add docstring

Reply:
+1 I think we should use more general name for the lock, because we may want to reuse it for different purposes later.